precompiles/randomness/src/tests.rs

```diff
@@ -49,27 +47,27 @@ use crate::Action;
 // 	assert!(selectors.contains(&(Action::ExecuteRequestExpiration as u32)));
 // }
 
-#[test]
-fn test_solidity_interface_has_all_function_selectors_documented_and_implemented() {
-	for file in ["Randomness.sol"] {
-		for solidity_fn in solidity::get_selectors(file) {
-			assert_eq!(
-				solidity_fn.compute_selector_hex(),
-				solidity_fn.docs_selector,
-				"documented selector for '{}' did not match for file '{}'",
-				solidity_fn.signature(),
-				file,
-			);
-
-			let selector = solidity_fn.compute_selector();
-			if Action::try_from(selector).is_err() {
-				panic!(
-					"failed decoding selector 0x{:x} => '{}' as Action for file '{}'",
-					selector,
-					solidity_fn.signature(),
-					file,
-				)
-			}
-		}
-	}
-}
+// #[test]
```

Comment:
why is this commented?

Reply:
Missing mock, and the Call enum needs concrete type parameters while Action didn't.